universal.c

```diff
@@ -61,7 +66,7 @@ S_isa_lookup(pTHX_ HV *stash, const char * const name, STRLEN len, U32 flags)
     /* A stash/class can go by many names (ie. User == main::User), so 
        we use the HvENAME in the stash itself, which is canonical, falling
        back to HvNAME if necessary.  */
-    our_stash = gv_stashpvn(name, len, flags);
+    our_stash = gv_stashsvpvn_cached(aTHX_ namesv, name, len, flags);
```

Comment:
The gv_stashsvpvn_cached() macro will already add the aTHX_

Reply:
I've updated the branch.